RedHat-UX/red-hat-design-system
# feat(tile): include grid layout in `rh-tile-group` element

## What I did

1. Removed CSS grid / flexbox layout from any demos that had an `<rh-tile-group>` element.
1. Added CSS grid to apply to the `:host` of `<rh-tile-group>`.
1. Improved the layout of the Color Context demo.
1. Removed the `tile-group-flex.html` demo.
1. Removed the `tile-layout.html` demo.
1. Set `<main>` to `display: block;` to improve layout in some `rh-tile` demos.
1. Removed `<rh-tile-group>` from the custom props demo.

## Testing Instructions

Check the DP of the following demos:

  * [Tile Group](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/tile-group/)
  * [Tile Group Radio](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/tile-group-radio/)
  * [Tile Group Radio Disabled](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/tile-group-radio-disabled/)
  * [Tile Group Disabled](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/tile-group-disabled/)
  * [Color Context](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/color-context/)
  * [Tile Group Custom Props](https://deploy-preview-1729--red-hat-design-system.netlify.app/elements/tile/demo/custom-props/)

## Notes to Reviewers

This PR does not add any grid/flexbox styles to the demos that don't have a `<rh-tile-group>` element in them. Currently, `<rh-tile>` styles are untouched. If `rh-tile` styles should implement a grid layout or max-width, let me know and I can make that change. 

Closes #1303.
